mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/7 - Quit to Main Menu

Closes #7 

To test:
- Open any scenario
- Open scenario details dialog button in upper left)
- Click Quit button
- Make sure you are on the main menu
- Try to play another scenario and make sure it works correctly
